test/test_helper.rb

```diff
@@ -1,3 +1,91 @@
 require "minitest"
 require 'simplecov'
 SimpleCov.start
+
+def lunch_data
+  [{ :district=>"Colorado",
+     :time_frame=>"2000",
+     :data_format=>"Percent",
+     :data=>"0.07",
+     :poverty_level=>"Eligible for Reduced Price Lunch" },
+   { :district=>"Colorado",
+     :time_frame=>"2000",
+     :data_format=>"Percent",
+     :data=>"0.27",
+     :poverty_level=>"Eligible for Free or Reduced Lunch" },
+   { :district=>"Colorado",
+     :time_frame=>"2000",
+     :data_format=>"Percent",
+     :data=>"0.2",
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.